app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -49,60 +51,233 @@ function DownloadButton({ bucketId, deFilePath }) {
+  const [sorting, setSorting] = React.useState(defaultSorting)
+  const [pagination, setPagination] = React.useState({
+    pageIndex: 0,
+    pageSize: numRows
+  })
+
+  const logProps = {
+    species, clusterName, annotation
+  }
+
+  const columns = React.useMemo(() => [
+    columnHelper.accessor('name', {
+      header: 'Name',
+      cell: deGene => {
+        return (
+          <label
+            title="Click to view gene expression.  Arrow down (↓) and up (↑) to quickly scan."
+          >
+            <input
+              type="radio"
+              name="selected-gene-de-table"
+              data-analytics-name="selected-gene-differential-expression"
+              value={deGene.getValue()}
+              onChange={event => {
+                deGene.table.resetRowSelection(deGene.row)
+                deGene.table.setRowSelection(deGene.row)
+
+                logProps.event = event
+                logProps.rank = deGene.i
+
+                searchGenesFromTable([deGene.getValue()], searchGenes, logProps)
+
+                deGene.row.getToggleSelectedHandler()
+              }}/>
+            {deGene.getValue()}
+          </label>
+        )
+      }
+    }),
+    columnHelper.accessor('log2FoldChange', {
+      header: () => (
+        <span
+          id="log2-fold-change-header"
+          className="glossary"
+          data-toggle="tooltip"
+          data-original-title="Log (base 2) of fold change">
+          log<sub>2</sub>(FC)
+        </span>
+      ),
+      cell: deGene => {
+        return deGene.getValue()
+      }
+    }),
+    columnHelper.accessor('pvalAdj', {
+      header: () => (
+        <span
+          id="pval-adj-header"
+          className="glossary"
+          data-toggle="tooltip"
+          data-original-title="p-value adjusted with Benjamini-Hochberg FDR correction">
+          Adj. p-value
+        </span>
+      ),
+      cell: deGene => {
+        return deGene.getValue()
+      }
+    })
+  ]
+  , [genesToShow]
+  )
+
+  const data = React.useMemo(
+    () => genesToShow,
+    [genesToShow]
+  )
+
+  const table = useReactTable({
+    columns,
+    data,
+    getCoreRowModel: getCoreRowModel(),
+    state: {
+      rowSelection,
+      sorting,
+      pagination
+    },
+    onRowSelectionChange: setRowSelection,
+    getSortedRowModel: getSortedRowModel(),
+    enableMultisort: true,
+    onPaginationChange: setPagination,
+    onSortingChange: setSorting,
+    getPaginationRowModel: getPaginationRowModel()
+  })
+
+  const dotPlotGenes = table.getPaginationRowModel().rows.slice(0, numRows).map(row => (
+    row.getAllCells().map(cell => {
+      return cell.getValue()
+    })[0]
+  ))
+
+  const verticalPad = 400 // Accounts for all UI real estate above table header
+  const tableHeight = window.innerHeight - verticalPad
+
   return (
     <>
-      <table className="de-table table table-terra table-scp-compact">
+      <div className="de-table-buttons">
```

Comment:
The grouping of these buttons and the tooltips you added are really nice! However, it makes the info icon feel a bit broken since hovering doesn't do anything on it, and you have to click to get the info. Would it make sense to add a tool tip to the info icon that says like "click here for extra info" so the functionality matches your new buttons?

Reply:
Nice catch, [done](https://github.com/broadinstitute/single_cell_portal_core/pull/1768/commits/e984bed78d6c583c751d9f857efd24b384f975e0#diff-e720c4093bef2a16eebc9f4600a30381f05b28a9d0e8231527680cac147477c2R27)!